feature/booth/src/main/kotlin/com/unifest/android/feature/booth/MenuImageDialog.kt

```diff
@@ -36,10 +37,11 @@ internal fun MenuImageDialog(
     menu: MenuModel,
     modifier: Modifier = Modifier,
 ) {
-    val configuration = LocalConfiguration.current
-    val dialogSize = remember(configuration) {
-        val screenWidth = configuration.screenWidthDp.dp
-        val screenHeight = configuration.screenHeightDp.dp
+    val density = LocalDensity.current
+    val windowInfo = LocalWindowInfo.current
+    val dialogSize = remember(windowInfo) {
+        val screenWidth = with(density) { windowInfo.containerSize.width.toDp() }
+        val screenHeight = with(density) { windowInfo.containerSize.height.toDp() }
         min(screenWidth, screenHeight) - 36.dp
```

Comment:
`with` 블록을 두 개 여는 것보다 하나의 블록에서 처리하는 것은 어떨까요 ?!
```suggestion
        with(density) { 
                val screenWidth = windowInfo.containerSize.width.toDp()
                val screenHeight = windowInfo.containerSize.height.toDp()
                min(screenWidth, screenHeight) - 36.dp
        }
```